### PR TITLE
Added missing plugin mechanism tests

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added or updated automated unit tests to verify correctness of my new code.
   * [ ] I've added or updated integration tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are working by executing CLI's commands manually.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
Added missing tests:
* Code from disabled plugins did not run
* Plugin don't override command

### CODE REVIEW

File: tests/test_data/configs/disabled_override_plugin_config.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[connections.test]
```
Comment (sfc-gh-pczajka) [inline tests/test_data/configs/disabled_override_plugin_config.toml:1]: Should we add the test for not-enabled plugin as well? That is:
```
[cli.plugin.something]
enabled = false
```

Comment (sfc-gh-astus) [reply tests/test_data/configs/disabled_override_plugin_config.toml:1]: This is that case `not in config` == disabled

**Review by sfc-gh-pczajka — DISMISSED** [review]

File: tests/plugin/test_broken_plugin.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from textwrap import dedent
+
+import pytest
+
+
+def test_broken_plugin(runner, test_root_path, _install_plugin, caplog):
```
Comment (sfc-gh-pjob) [inline tests/plugin/test_broken_plugin.py:6]:
```suggestion
def test_broken_command_path_plugin(runner, test_root_path, _install_plugin, caplog):
```

File: tests/plugin/test_broken_plugin.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+from textwrap import dedent
+
+import pytest
+
+
```
Comment (sfc-gh-pjob) [inline tests/plugin/test_broken_plugin.py:5]: as discussed let's add test case `test_broken_implementation_plugin` which will try to load a plugin which raises some error during its module loading

File: tests/plugin/test_override_by_external_plugins.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        """+---------------------------------------+
+| connection_name | parameters          |
+|-----------------+---------------------|
+| test            | {'account': 'test'} |
++---------------------------------------+
+    """
+    )
+
+
+def test_disabled_plugin_is_not_executed(
```
Comment (sfc-gh-pjob) [inline tests/plugin/test_override_by_external_plugins.py:27]: as discussed let's add one more test case confirming that no code from disabled plugin's module is executed

we need to check that not only command implementation is not executed but we need to be sure that no code is executed

File: tests/plugin/test_broken_plugin.py (hunk 3 above)
Comment (sfc-gh-astus) [reply tests/plugin/test_broken_plugin.py:5]: Named it `failing_plugin`

**Review by sfc-gh-pjob — APPROVED** [review]

**Pull request merged**